static/app/views/issueDetails/traceTimeline/traceTimeline.tsx

```diff
@@ -35,14 +33,17 @@ export function TraceTimeline({event}: TraceTimelineProps) {
     !isLoading && data.length > 0 && data.every(item => item.id === event.id);
   if (isError || noEvents || onlySelfEvent) {
     // display empty placeholder to reduce layout shift
-    return <div style={{height: PLACEHOLDER_SIZE}} data-test-id="trace-timeline-empty" />;
+    return <div style={{height: 34}} data-test-id="trace-timeline-empty" />;
```

Comment:
smaller overall

Reply:
should we keep this value in a variable since it's being re-used on line 93 as well?